internal/handlers/handlers.go

```diff
@@ -171,21 +169,19 @@ func (h *Handler) ProcessVerificationResult() fiber.Handler {
 		)
 		sigHeader := c.Get("Idenfy-Signature")
```

Comment:
extract the headers that we use frequently into consts to avoid typos when reusing them

Reply:
Done